doc/man3/EVP_des_cbc.pod

```diff
@@ -54,6 +54,9 @@ EVP_des_ofb()
 DES in CBC, ECB, CFB with 64-bit shift, CFB with 1-bit shift, CFB with 8-bit
 shift and OFB modes.
 
+All these algorithms are available only when the legacy provider is loaded or
```

Comment:
What about: _only when the legacy provider or a third part implementation is loaded_ instead?

Reply:
(nit) please spell it "third-party implementation" (throughout)